public/components/main/reports_table.tsx

```diff
@@ -95,7 +96,7 @@ export function ReportsTable(props) {
     setShowLoading(e);
   };
 
-  const onDemandDownload = async (id: any) => {
+  const onDemandDownload = async (id: any, item: any) => {
```

Comment:
Try to typed this prop

Reply:
This comes from a prop that already comes from the reporting repository and has as type any